docs/components/integration-framework/connectors/out-of-the-box-connectors/rest.md

```diff
@@ -6,6 +6,9 @@ description: Make a request to a REST API and use the response in the next steps
 
 The **REST Connector** allows you to make a request to a REST API and use the response in the next steps of your process.
 
+Find the source code of this Connector in our [GitHub repository](https://github.com/camunda/connector-http-json).
```

Comment:
Proposal: Instead of being verbose here, be quick and concise. Off-load essential information (how to run/consume this) to GitHub.

```
Checkout the connector's sources as well as self-managed run-instructions on [GitHub](https://github.com/camunda/connector-http-json).
```

Reply:
I personally like self-contained documentation more than scattered documentation. As a reader, I don't want to jump to GitHub just to read how I can do this on SM. But that's just me, maybe.

Reply:
I think it's important to have a short description of what the Connector actually does that users can easily understand in one glance, without asking them to jump to other links if it's not necessary. This was a complaint we got from Josh, as a user wanting to accomplish one task, we ask them to go to multiple different areas of our documentation which aren't easy to find. 
We provide a short concise description for all other Connectors which is very helpful.

Reply:
Thanks for your feedback. Agreed on being self-contained. We just have to be aware of different users "reading" our documentation, and mixing these different concerns may confuse them:

* Is this user documentation
* Is this self-managed deployment documenentation

Taking my proposal we'd clearly be able to separate the audiences => You find up to date "run yourself" instructions on GitHub, and user guide ("using connector OOTB, in our modelers") in the docs.

Reply:
Also, I think we could create a new "Source Code" section at the bottom of the page of the Connectors where users can go, instead of highlighting it within the description and at the very top.

Reply:
> Also, I think we could create a new "Source Code" section at the bottom of the page of the Connectors where users can go, instead of highlighting it within the description and at the very top.

Absolutely agreed. (Modeling) user guide should be the primary focus of that documentation.

Reply:
A dedicated section "run in self-managed" at the bottom of every page would be awesome and would clearly separate the different reads.

Reply:
Done with [52ceaf2](https://github.com/camunda/camunda-platform-docs/pull/1229/commits/52ceaf21811d6079eadc05345339f31eabbcbe57)